.github/workflows/on_pull_request.yml

```diff
@@ -418,6 +418,11 @@ jobs:
         run: yarn test:website
         env:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
+          APPLITOOLS_BATCH_NAME: "${{ github.event.pull_request.head.repo.full_name }}: pull request from ${{ github.event.pull_request.head.ref }} - ${{ github.event.pull_request.title }} (${{ github.event.pull_request.head.sha }})"
+          GITHUB_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+          GITHUB_BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
+          USE_CYPRESS_EYES: true
```

Comment:
❓ Saw some folks using a similar technique to conditionally run/not run the applitools checks and thought it could be useful. What do folks think?

Reply:
Why would we conditionally run them?

Reply:
Except for the purposes of developing on or debugging the applitools configuration and tests, we do not need to run them locally. This allows us a simple way to run or skip all applitools tests.